dbtsl/api/graphql/client/asyncio.py

```diff
@@ -31,6 +31,7 @@ def __init__(
         environment_id: int,
         auth_token: str,
         url_format: Optional[str] = None,
+        timeout_ms: Optional[int] = None,
```

Comment:
I think we should define timeouts in seconds, not ms. There's no real reason to ever have a timeout be less than 1s.